### ISSUE TITLE
System Upgrade Chips Removed From Game. 

### ISSUE THREAD

**Issue by Jigan5** [tsunamayo/Starship-EVO#4783]
### Short Description of the issue:

System Upgrade chips removed from game. Ships that had chips installed are reset with no chips in inventory. Destroyed vessels no longer drop system chips. Bounty contracts no longer award system chips. 

### Steps to reproduce the issue:

Unknown, persistent issue.

### Environment
- Starship EVO version: 22w301 steam
- Graphics Card: GeForce 2080TI

### Additional information:

Revalidated steam files several times. Created new save to test. Still issue persists. 

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/9224054/Player.log)

**Comment by tsunamayo** [comment]
@Jigan5 Yes indeed I should have mentioned this, they are removed for now as things are changing on the item drop side, and I want system chips to use the new framework. They will be back! Thanks
